pkg/test/utils/kubernetes.go

```diff
@@ -321,7 +321,7 @@ func Namespaced(dClient discovery.DiscoveryInterface, obj runtime.Object, namesp
 	resource, err := GetAPIResource(dClient, obj.GetObjectKind().GroupVersionKind())
 	if err != nil {
 
-		return "", "", err
+		return "", "", fmt.Errorf("retrieving API resource for %v failed: %v", obj.GetObjectKind().GroupVersionKind(), err)
```

Comment:
perhaps the 2nd `%v` should be a `%w` 
```suggestion
		return "", "", fmt.Errorf("retrieving API resource for %v failed: %w", obj.GetObjectKind().GroupVersionKind(), err)
```